feedthefox/users/providers/github/urls.py

```diff
@@ -0,0 +1,6 @@
+from allauth.socialaccount.providers.oauth2.urls import default_urlpatterns
+
+from .provider import FeedTheFoxGitHubProvider
```

Comment:
Is there a reason you are using relative imports here? Generally this is not good practice.